test/console_TEST.cc

```diff
@@ -1,3 +1,37 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2020, Willow Garage, Inc.
```

Comment:
This year and company name combination doesn't make any sense. Please check the file history for the correct info.